clang/test/utils/update-verify-tests/Inputs/duplicate-diag.c

```diff
@@ -1,3 +1,5 @@
+// RUN: %clang_cc1 -verify %s
+// RUN: diff %s %s.expected
```

Comment:
I still think those new invocations of `diff` should be in `lit-plugin.test`.

Those `diff` lines attempt to check that this file (or rather its temporarily copied version) was overwritten correctly.

It isn't sound to have the code to check whether a file was correctly overwritten in the file that is overwritten.